src/site/content/en/metrics/lcp/index.md

```diff
@@ -102,9 +102,10 @@ considered for Largest Contentful Paint are:
 - [Block-level](https://developer.mozilla.org/docs/Web/HTML/Block-level_elements)
   elements containing text nodes or other inline-level text elements children.
 
-Note, restricting the elements to this limited set was intentional in order to
-keep things simple in the beginning. Additional elements (like the full `<svg>` support or
-`<video>` elements without poster images) may be added in the future as more research is conducted.
+Additionally, the following elements will be considered for a limited set of users as of version 116 of Chrome, but will be considered for all users as of version 117:
```

Comment:
Can you confirm with Ian that this is going to be true? I didn't get the impression that restricting the finch trial to just M116 was something we had all agreed to do.

Reply:
Also, it's probably worth waiting on this until after the change is published on the metric changelog, and then we could link to the changelog from whatever note we add here.

Reply:
That's all fair—I felt a need to get ahead of this a bit, so we could deploy this change as soon as we felt we were in a good position to do so.

cc: @clelland